.github/workflows/test.yml

```diff
@@ -30,4 +30,9 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
 
-      - run: go test ./... -v -race -count=1
+      - name: Setup gotestsum
+        uses: autero1/action-gotestsum@v2.0.0
```

Comment:
This action hasn't been updated since early 2023. See this https://github.com/autero1/action-gotestsum/issues/46

Reply:
I think we better of by running `go install gotest.tools/gotestsum@latest`

Reply:
Yeah we can also do `go run gotest.tools/gotestsum@v1.11.0`, it just might take a little bit to build on the windows & darwin runners.

But the action is really just a fancy `curl -L https://github.com/gotestyourself/gotestsum/releases/download/v${VERSION}/gotestsum_${VERSION}_${GOOS}_${GOARCH}.tar.gz | tar -xz` so I suppose I could do that instead

Reply:
That's too cumbersome, just use `go install`, we already do that for `gosec`and `govulncheck`.